TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Shared/_CheckboxFilter.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@model CheckboxFilterViewModel
+
+<govuk-checkboxes id-prefix="trs-filters-@Model.Name" name=@Model.Name checkboxes-class="govuk-checkboxes govuk-checkboxes--small">
+    <govuk-checkboxes-fieldset>
+        <govuk-checkboxes-fieldset-legend is-page-heading=true class="govuk-fieldset__legend--s">
+            @Model.DisplayName
+        </govuk-checkboxes-fieldset-legend>
```

Comment:
```suggestion
        <govuk-checkboxes-fieldset-legend is-page-heading=true class="govuk-fieldset__legend--s" />
```

Reply:
@gunndabad I don't understand this change - where would it get the text for the legend from if I don't pass it in?

Reply:
You're right - as it's not using model binding it'll have to be specified this way